packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -706,19 +705,22 @@ public Model runAlignAMR(
 			provenanceService.createProvenance(provenance);
 
 			// update model embeddings
-			if (card != null && model.getPublicAsset() && !model.getTemporary()) {
-				final String cardText = objectMapper.writeValueAsString(card);
-				if (cardText != null) {
-					try {
-						final TerariumAssetEmbeddings embeddings = embeddingService.generateEmbeddings(cardText);
 
-						modelService.uploadEmbeddings(modelId, embeddings, hasWritePermission);
-						notificationInterface.sendMessage("Embeddings created");
-					} catch (final Exception e) {
-						log.warn("Unable to generate embedding vectors for model");
-					}
-				}
-			}
+			// TODO: do you want to update the model asset embeddings in the project index?
```

Comment:
These weren't used anymore, but if you want the model embeddings updated after extraction in the project index, here is where to put it.